components/math/multiplicationConnect/GameBoardBlock.tsx

```diff
@@ -1,20 +1,20 @@
 import React, {useState} from 'react'
+import { calculateWinner } from '../../../pages/studentPortal/labs/multiplication-connect/Index';
 
 
-const blockClick = (block) => {
-    (block.isSelected===false) ? block.isSelected=true : block.isSelected=false;
-    // console.log(block);
+const blockClick = (block, grid) => {
```

Comment:
Can you give types to these input parameters?

Reply:
Yeah, I'm going to add the typescript throughout after I push this PR out